spec/pushmi_pullyu/logging_spec.rb

```diff
@@ -80,12 +87,26 @@ class LoggerTest
     end
   end
 
-  describe '.log_preservation_event' do
+  describe '.log_preservation_success' do
     let(:tmp_log_dir) { 'tmp/logs' }
     let(:tmp_aip_dir) { 'tmp/test_aip_dir' }
 
-    it 'logs preservation event to both preservation log and application log' do
+    before do
+      FileUtils.mkdir_p(tmp_aip_dir)
+      FileUtils.mkdir_p(tmp_log_dir)
+      Timecop.freeze(Time.now)
+    end
+
+    after do
+      FileUtils.rm_rf(tmp_aip_dir)
       FileUtils.mkdir_p(tmp_log_dir)
+      Timecop.return
+    end
+
+    it 'logs preservation event to both preservation log and application log' do
+      # Make sure we are initialize logger with expected file destinations and not default stdout destination
+      PushmiPullyu::Logging.initialize_loggers(events_target: "#{tmp_log_dir}/preservation_events.log",
```

Comment:
I wonder if tagged logging could be used rather than two multiple log files/loggers? (https://api.rubyonrails.org/classes/ActiveSupport/TaggedLogging.html) I suppose it would depend on how the logs are used to troubleshoot and the content of the logs itself. I think how the logs are initialised are fine however and everything else looks good to me.

Reply:
That is a good point @ConnorSheremeta . Having different loggers was done before this work was done. I believe the idea is to have the logs handled separately so they can be parsed based on need.

I will keep in mind your suggestion if we want to simplify logging at a later date.